supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -143,6 +143,24 @@ $ oc describe node __<node_name>__
 ----
 ====
 
+
+[[italics-double-quotation-marks]]
+== Italics and double quotation marks
+
+Ensure that you use italics when introducing or defining a term, and use double quotation marks ("") when emphasizing how to write a term. This practice provides better stylistic consistency on how Red Hat communicates terminology to Red Hat customers.
+
+.Examples of defining terms in documentation
+
+* A _host group_ is a group of one or more hosts. 
+* _Comma-separated_ values are a set of values in which each value is separated by a comma.
```

Comment:
```suggestion
* _Comma-separated values_ are a set of values in which each value is separated by a comma.
```
Shouldn't the entire phrase be italicized here?

Reply:
Hi @bburt-rh . Good catch! Updated.